src/main/java/Calculator/com/Dmitriev/CalculatorController.java

```diff
@@ -0,0 +1,46 @@
+
+
+package Calculator.com.Dmitriev;
+
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/calculator")
+public class CalculatorController {
+
+    private final CalculatorService calculatorService;
+
+    public CalculatorController(CalculatorService calculatorService) {
+        this.calculatorService = calculatorService;
+    }
+
+
+    @GetMapping()
+    public String Hello() {
+        return "Добро пожаловать в калькулятор!";
+    }
+
+    @GetMapping("/plus")
+    public String getSumma(Integer a, Integer b) {
+        return (a + "+" + b + "=" + (a+b));
```

Comment:
Сервис у тебя нигде не используется. Заинжектил и все. А в методе контроллера должен быть его вызов
```suggestion
        return (a + "+" + b + "=" + calculatorService.getSumma(a,b));
```